Makefile

```diff
@@ -124,6 +124,11 @@ token-propagation-integration-test:
 	go clean -testcache
 	bash -c "set -e; set -o pipefail; $(GOTEST) -tags token_propagation -run TestBearTokenPropagation $(STORAGE_PKGS) | $(COLORIZE)"
 
+.PHONY: kafka-kerberos-integration-test
+kafka-kerberos-integration-test:
```

Comment:
I assume you'd want to call it somewhere from .travis.yaml ?

Reply:
Yes, that is the intention.